config/checkstyle/checkstyle.xml

```diff
@@ -184,7 +184,7 @@
             <property name="braceAdjustment" value="0"/>
             <property name="caseIndent" value="0"/>
             <property name="throwsIndent" value="4"/>
```

Comment:
![image](https://user-images.githubusercontent.com/60384540/97529242-728fcb00-19ea-11eb-9bb6-62ae0413826a.png)
The arrow coverges too much in the circled portion and thus it is making the class diagram more complicated than it should have been.  Maybe try to simplify the diagram by breaking it into portions or structure the arrows in a way that it is straight and pointing the class table more clearly.